.github/workflows/go.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.13
+          go-version: 1.14
```

Comment:
why not 1.15 ? Just wondering ...

Reply:
If we use 1.15 we are on the latest version which might restrict potential users because they have to upgrade.
I have upgraded to 1.14 because of a feature needed in the logger, else I would not upgrade at all to keep more versions compatible.